libraries/AP_HAL_ChibiOS/hwdef/F4BY_H743/README.md

```diff
@@ -0,0 +1,192 @@
+
+![F4BY_H743](F4BY_H743_board_image.jpg)
+
+
+
+Where to Buy
+============
+
+Shop `here https://f4by.com/en/?order/our_product`__
+The instructions, schematic, 3D model  are available `here https://f4by.com/en/?doc/fc_f4by_v3.0.1_h743`__
+
+Specifications
+==============
+    - Processor
+
+        - STM32H743 32-bit processor, 480Mhz
+        - 2MB Flash
+        - 1MB RAM
+
+    Sensors
+        - ICM-42688-P IMU (F4BY_H743 v3.0.3)  
+        - IIM-42652   IMU (F4BY_H743 v3.0.2)
+        - MPU6000     IMU (F4BY_H743 v3.0.1)   
+        - MS5611 Barometer 
+        - QMC5883P Compass   
+    Power
+        - Power from external power module (range +5.0 ... +6.0V)
+        - Anslog current and voltage analog inputs (range 0 ... +6,6V)
+    Interfaces
+        - 16x PWM outputs DShot capable (include 4x BDShot)
+        - 1x RC input
+        - 5x UARTs/serial for GPS, bi-directional RC and other peripherals)
+        - 1x I2C port for external compass, airspeed, etc.
+        - 1x CAN bus
+        - Buzzer output
+        - micro SD Card for logging, Lua scripts etc.
+        - USB type-C port
+        - USB GHS type port for external connector.
+        - DFU bootloader button
+
+
+Analog sensors
+==============
+
+- Analog airspeed PC0 (0...+3.3v range) pin 10
+- Analog rssi PC1     (0...+3.3v range) pin 11
+- Voltage sensing PC3 (0...+6.6v range) pin 13
+- Current sensing PC2 (0...+6.6v range) pin 12
+- Board power sensing PC4 (0...+6.6v range)
+- Servo power sensing PC5 (0...+9.9v range)
+
+Analog Airspeed
+===============
+If the ARSPD pin is used for analog airspeed  input. 
+Set :ref:`ARSPD_PIN<ARSPD_PIN>` to 10. 
+Set :ref:`ARSPD_TYPE<ARSPD_TYPE>` to "1".
+
+Analog RSSI
+===========
+If the RSSI pin is used for analog RSSI input. Set :ref:`RSSI_ANA_PIN<RSSI_ANA_PIN>` to 11. 
+Set :ref:`RSSI_TYPE<RSSI_TYPE>` to "1" .
+
+
+UART Mapping
+============
+The UARTs are marked Rn and Tn in the above pinouts. The Rn pin is the receive pin for UARTn. The Tn pin is the transmit pin for UARTn
+
+|Port  | UART    |Protocol        |TX DMA |RX DMA |
+|------|---------|----------------|-------|-------|
+|0     |  USB    |  MAVLink2      |  ✘    |   ✘   |
+|1     |  USART2 |  RC            |  ✔    |   ✔   |
+|2     |  UART1  |  MAVLink2      |  ✔    |   ✔   |
+|3     |  USART3 |  GPS           |  ✔    |   ✘   |
+|4     |  UART4  |  None          |  ✔    |   ✘   |
+|5     |  UART5  |  None          |  ✔    |   ✘   |
+
+
+RC Input
+========
+
+- RCin  PB0  
+Using the RCin pin will support all unidirectional RC protocols. (PPM, SBUS, iBus, PPM-Sum, DSM,DSM2,DSM-X,SRXL and SUM-D)
+
+- USART2 for Bi-directional protocols (CRSF/ELRS,SRXL2,IRC Ghost, and FPort) see `here <https://ardupilot.org/sub/docs/common-rc-systems.html#common-rc-systems>`
+
+PWM Output
+==========
+
+PWM/DShot capable motor outputs:
+
+Group #1
+
+|Out   | Port    |GPIOPIN |BDSHOT |
+|------|---------|--------|-------|
+|1     | PA0     | 50     | ✔     |
```

Comment:
These are not correct. They all support BDSHOT because the support is in pairs - so you should tick 1-8